hs_core/views/__init__.py

```diff
@@ -168,7 +168,7 @@ def update_key_value_metadata(request, shortkey, *args, **kwargs):
         err_message = ex.message
 
     if is_update_success:
-        resource_modified(res, request.user)
+        resource_modified(res, request.user, overwrite_bag=False)
```

Comment:
This is a little confusing. Why is the bag not overwritten in this case? Is it because the metadata isn't part of the bag?